exonum/src/api/public/system.rs

```diff
@@ -31,6 +32,12 @@ pub struct HealthCheckInfo {
     pub connectivity: bool,
 }
 
+#[doc(hidden)]
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
+pub struct ConsensusStatusInfo {
+    pub status: bool,
```

Comment:
`status: true` and `status:false` are not really informative. Maybe use enum instead?

Reply:
We are using `true` and `false` in `HealthCheckInfo` too.

Reply:
Maybe change it there too then? 😈

Reply:
https://jira.bf.local/browse/ECR-1665